exporter/otlp/lib/opentelemetry/exporter/otlp/exporter.rb

```diff
@@ -187,6 +187,7 @@ def send_bytes(bytes, timeout:) # rubocop:disable Metrics/AbcSize, Metrics/Cyclo
             when Net::HTTPBadRequest, Net::HTTPClientError, Net::HTTPServerError
               # TODO: decode the body as a google.rpc.Status Protobuf-encoded message when https://github.com/open-telemetry/opentelemetry-collector/issues/1357 is fixed.
```

Comment:
Note that the issue referenced was fixed almost a year ago. We should be able to decode the body and potentially use information it contains in the `reason` label below.

Reply:
Tracking here https://github.com/open-telemetry/opentelemetry-ruby/issues/927